c#/protocol_combined/win64/protocol_combined/dynamixel.cs

```diff
@@ -21,9 +21,9 @@
 
 namespace dynamixel_sdk
 {
-  class dynamixel
+  unsafe class dynamixel
   {
-    const string dll_path = "../../../../../../../../c/build/win64/output/dxl_x64_c.dll";
+    const string dll_path = "dxl_x64_c.dll";
```

Comment:
@elliotwoods 

Did you touch some environment settings, and let the path as absolute path?

Reply:
No, I just copy the dll into the output directory